### DISCUSSION TITLE
Installing larastan package too strict

### DISCUSSION THREAD

**Discussion by sbaum2s** [larastan/larastan#2003] [General]
From the readme

`composer require larastan/larastan:^2.0 --dev`

fails with error message

```
larastan/larastan 2.0.0 requires illuminate/console ^9 -> found illuminate/console[v9.0.0-beta.1, ..., 9.x-dev] but these were not loaded, likely because it conflicts with another require.
```

But how to find out what it conflicts with?

At the end of the story everything runs well with

`composer require larastan/larastan --dev`

It pulls the latest version and that´s it.

**Comment by szepeviktor** [comment]
Hello @sbaum2s 👋🏻
You must be on Windows.

Duplicate of https://github.com/larastan/larastan/issues/1971

**Reply by calebdw** [reply]
This comes up often enough that it may be worth a note in the readme?

**Reply by szepeviktor** [reply]
2f34808 👀

**Reply by sbaum2s** [reply]
> Hello @sbaum2s 👋🏻 You must be on Windows.
> 
> Duplicate of #1971

Yes i am on Windows, and the readme still contains the strict version of the command. Other than that the search here did not show me the other issue.

**Reply by calebdw** [reply]
It's been wrapped in a string
